pom.xml

```diff
@@ -284,7 +285,7 @@
         <gravitee-reactor-message.version>5.0.0-alpha.11</gravitee-reactor-message.version>
         <gravitee-reactor-native-kafka.version>1.0.0-alpha.23</gravitee-reactor-native-kafka.version>
         <gravitee-apim-repository-bridge.version>5.1.2</gravitee-apim-repository-bridge.version>
-        <gravitee-secretprovider-hc-vault.version>1.0.0</gravitee-secretprovider-hc-vault.version>
+        <gravitee-secretprovider-hc-vault.version>2.0.0-APIM-7497-api-secrets-SNAPSHOT</gravitee-secretprovider-hc-vault.version>
```

Comment:
### :warning: Aqua detected sensitive data in your code  
**Category:** Config Generic   
**Description:** Generic Config Credential   
**Severity:** HIGH   
**Match:** cretprovider-hc-vault.version>************************************</gravitee-secretpro  
[This comment was created by Aqua Pipeline]